src/widgets/study-with-us/study-with-us.test.tsx

```diff
@@ -23,7 +23,7 @@ describe('School Component', () => {
   it('renders the study options correctly', () => {
     const options = screen.getAllByRole('heading', { level: 3 });
 
-    expect(options.length).toBe(3);
+    expect(options.length).toBe(4);
```

Comment:
I'm not sure we should test it like this - it's a very inflexible test.
@SpaNb4